source

```diff
@@ -100164,255 +100161,256 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
+      not separated by an <span>ASCII whitespace</span> character. In this case the parser behaves as if
       a whitespace character is present.</p>
     <tr>
       <td>
       <dfn data-x="parse-error-unexpected-character-after-doctype-system-identifier">unexpected-character-after-doctype-system-identifier</dfn>
       <td><p>This error occurs if the parser encounters any characters other than <span>ASCII
       whitespace</span> characters after the <span data-x="syntax-doctype">DOCTYPE</span> system
-      identifier. Parser ignores these characters.</p>
+      identifier. The parser ignores these characters.</p>
     <tr>
       <td>
       <dfn data-x="parse-error-eof-in-cdata">eof-in-cdata</dfn>
       <td><p>This error occurs if the parser encounters the end of the <span>input stream</span> in
-      a <span data-x="syntax-cdata">CDATA section</span>. Parser treats such CDATA section as if it
-      is closed right before the end of the input stream.</p>
+      a <span data-x="syntax-cdata">CDATA section</span>. The parser treats such CDATA sections as if they
+      are closed immediately before the end of the input stream.</p>
     <tr>
       <td>
       <dfn data-x="parse-error-missing-semicolon-after-character-reference">missing-semicolon-after-character-reference</dfn>
       <td><p>This error occurs if the parser encounters a <span data-x="syntax-charref">character
-      reference</span> that is not terminated by a U+003B SEMICOLON (;) character. Usually parser
-      behaves as if character reference is terminated by the U+003B SEMICOLON (;) character, however
-      there are some ambigious cases in which parser includes subsequent characters in a character
+      reference</span> that is not terminated by a U+003B SEMICOLON (;) character. Usually the parser
+      behaves as if character reference is terminated by the U+003B SEMICOLON (;) character; however,
+      there are some ambiguous cases in which the parser includes subsequent characters in the character
       reference.</p>
 
       <p class="example">For example, <code data-x="">&amp;not;in</code> will be parsed as
-      "<code data-x="">&not;in</code>", whereas <code data-x="">&amp;notin</code> will be parsed as
+      "<code data-x="">&not;in</code>" whereas <code data-x="">&amp;notin</code> will be parsed as
       "<code data-x="">&notin;</code>".</p>
     <tr>
       <td>
       <dfn data-x="parse-error-unknown-named-character-reference">unknown-named-character-reference</dfn>
       <td><p>This error occurs if the parser encounters an <span
-      data-x="syntax-ambiguous-ampersand">ambiguous ampersand</span>. In that case parser doesn't
-      resolve the <span data-x="syntax-charref">character reference</span>.</p>
+      data-x="syntax-ambiguous-ampersand">ambiguous ampersand</span>. In this case the parser doesn't
+      resolve the <span data-x="syntax-charref">character reference</span>.</p><!-- TODO: does this mean it's treated as text content? -->
```

Comment:
It can be treated both as text content or as a part of attribute value, depending on the position where character reference was consumed.